Rakefile

```diff
@@ -39,6 +39,38 @@ task :rubygems_admins do
   end
 end
 
+task :update_dependencies do
+  puts "Updating all internal fastlane dependencies"
+
+  regex = %r{spec.add_dependency .(.+).\, .\>\= (\d+\.\d+\.\d+).\, .\< \d+\.\d+\.\d+.}
```

Comment:
Currently only works with 3 parts, which is the notation we've used until now. Should we keep this assumption?